defaults/main.yml

```diff
@@ -81,6 +81,14 @@ firewall_ports:
   - 9090/tcp
   - 16514/tcp
 
+# Define additional firewall ports to be exposed here
+firewall_ports_custom:
+#  - 1234/tcp
+
+# Define additional firewall services to be exposed here
```

Comment:
This might be one of the rare use cases where we should stash the required ports and services into vars/main.yml.

Keep custom_firewall_X in defaults and the base ones over in vars/main.yml and should be good.